src/Reddit/README.md

```diff
@@ -14,6 +14,8 @@ Please see the [Base Installation Guide](https://socialiteproviders.com/usage/),
 'reddit' => [    
   'client_id' => env('REDDIT_CLIENT_ID'),  
   'client_secret' => env('REDDIT_CLIENT_SECRET'),  
+  'response_type' => env('REDDIT_RESPONSE_TYPE', 'code'),
+  'state' => env('REDDIT_STATE', 'r@nd0m5tr1n6'),
```

Comment:
This is not how state works, it should be randomly generated each request. Have you told the provider to be stateless? Otherwise it should be generated automatically.

Reply:
Yeah, I'm doing stateless.